prime-router/docker-compose.yml

```diff
@@ -13,6 +13,11 @@ services:
         source: ./target
         target: /prime-data-hub-router
     entrypoint: [/prime-data-hub-router/start_func.sh]
+    env_file:
+      # On first run of docker-compose, create this file, it will get populated, but docker-compose doesn't allow optional env_file
+      # Example: touch .vault/env/.env.local
+      # https://github.com/docker/compose/pull/3955
+      - ./.vault/env/.env.local
```

Comment:
This bug is really annoying. If this file doesn't exist docker-compose does not start. It's a simple work around to creating the file (and it only needs to be done once), but I haven't found a great solution around this issue that also ensures we don't accidentally check in our local Vault keys.